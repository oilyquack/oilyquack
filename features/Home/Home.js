import React from "react";

import ollie from "./assets/ollie.png";
import styles from "./Home.module.css";

const Home = () => (
  <>
    <div className={styles.content}>
      <h1>Hi. My name is Ollie.</h1>
      <h2>I’m a software developer, noisy musician and cat enthusiast.</h2>
      <p>
        I like building great front end applications with a passion for making
        accessible web experiences.
      </p>
      <p>
        <strong>oilyquack</strong> is a{" "}
        <a
          href="https://en.wikipedia.org/wiki/Malapropism"
          target="_blank"
          rel="noopener noreferrer"
        >
          malapropism
        </a>{" "}
        of my real name. I started using it when I was a teacher to stop
        students finding me online. I’m too fond of how silly it sounds to
        change it back. I also get a kick out of people saying it out loud in
        job interviews.
      </p>
      <p>
        You can see some of my work on{" "}
        <a
          href="https://github.com/oilyquack"
          target="_blank"
          rel="noopener noreferrer"
        >
          GitHub
        </a>{" "}
        or read my ramblings on{" "}
        <a
          href="https://twitter.com/oilyquack"
          target="_blank"
          rel="noopener noreferrer"
        >
          Twitter
        </a>
        .
      </p>
    </div>
    <img alt="Ollie Crook" className={styles.image} src={ollie} />
  </>
);

export default Home;
