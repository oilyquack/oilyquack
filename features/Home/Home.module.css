.content {
  width: 100%;
}

.image {
  width: 100%;
}

@media (min-width: 768px) {
  .content {
    width: 75%;
  }

  .image {
    width: 75%;
  }
}
