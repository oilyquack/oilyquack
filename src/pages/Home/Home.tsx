import type { Component } from "solid-js";

import ollie from "./assets/ollie.jpg";
import styles from "./Home.module.css";

const Home: Component = () => {
  return (
    <>
      <div class={styles.content}>
        <h1>Hey there, I’m Ollie.</h1>
        <h2>
          I’m a software developer who loves creating apps that are both
          high-performing and easy to use.
        </h2>
        <p>Besides coding, I’m into making music and hanging out with cats.</p>
        <p>
          My online name, <strong>oilyquack</strong>, is a
          <a
            href="https://en.wikipedia.org/wiki/Malapropism"
            target="_blank"
            rel="noopener noreferrer"
          >
            malapropism
          </a>{" "}
          I started using as a way to stay incognito as a teacher, but now I
          just love how goofy it sounds.
        </p>
        <p>
          I work on all parts of a project, making sure things run smoothly and
          helping teams grow. Check out my code on{" "}
          <a
            href="https://github.com/oilyquack"
            target="_blank"
            rel="noopener noreferrer"
          >
            GitHub
          </a>
          . I'm passionate about what I do, and I'm excited to bring that
          enthusiasm to your next project.
        </p>
      </div>
      <img alt="Ollie Crook" class={styles.image} src={ollie} />
    </>
  );
};

export default Home;
