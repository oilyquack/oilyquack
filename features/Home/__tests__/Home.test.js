import { render } from "@testing-library/react";
import React from "react";

import Home from "../Home";

describe("features : Home", () => {
  it("renders correctly", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
