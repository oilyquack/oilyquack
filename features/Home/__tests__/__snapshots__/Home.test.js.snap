// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`features : Home renders correctly 1`] = `
<div>
  <div
    class="content"
  >
    <h1>
      Hi. My name is Ollie.
    </h1>
    <h2>
      I’m a software developer, noisy musician and cat enthusiast.
    </h2>
    <p>
      I like building great front end applications with a passion for making accessible web experiences.
    </p>
    <p>
      <strong>
        oilyquack
      </strong>
       is a
       
      <a
        href="https://en.wikipedia.org/wiki/Malapropism"
        rel="noopener noreferrer"
        target="_blank"
      >
        malapropism
      </a>
       
      of my real name. I started using it when I was a teacher to stop students finding me online. I’m too fond of how silly it sounds to change it back. I also get a kick out of people saying it out loud in job interviews.
    </p>
    <p>
      You can see some of my work on
       
      <a
        href="https://github.com/oilyquack"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub
      </a>
       
      or read my ramblings on
       
      <a
        href="https://twitter.com/oilyquack"
        rel="noopener noreferrer"
        target="_blank"
      >
        Twitter
      </a>
      .
    </p>
  </div>
  <img
    alt="Ollie Crook"
    class="image"
    src="test-file-stub"
  />
</div>
`;
